tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -149,6 +150,13 @@ def keyboard(self):
         from gaiatest.apps.keyboard.app import Keyboard
         return Keyboard(self.marionette)
 
+    @property
+    def manifest_url(self):
+        return '{}{}.{}/manifest.webapp'.format(self.DEFAULT_PROTOCOL, format(self.__class__).lower(), self.DEFAULT_APP_HOSTNAME)
```

Comment:
It turns out you don't need to do that, just fix the name issue and not make the change in gaia_test.py fixes it and launches the apps in tests correctly.